cmake/flash/CMakeLists.txt

```diff
@@ -1,93 +1,80 @@
 # SPDX-License-Identifier: Apache-2.0
 
+add_custom_target(flash_runner_property_target)
```

Comment:
There is already a target named `flash`, so are there any advantages to create yet another target called `flash_runner_property_target` instead of setting the properties directly on the `flash` target.

Reply:
I guess not, will change.

Reply:
The `flash` target is not defined at the time of running this cmake code, and moving the contents of this file around so that I have access to it is something I want to avoid. So I`m changing it to `runner_props`

Reply:
Just did a small re-ordering here, and initial testing deems this is safe.
The `flash`, `debug`, etc. targets depends on nothing prior in the CMakeLists.txt file, so the reordering should be safe.

But if you feel uneasy about it, we can take your PR in, and re-order at later point in time.
https://github.com/hakonfam/zephyr/pull/23

Reply:
I feel uneasy about it xD